decidim-core/lib/decidim/nicknamizable.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This concern contains the logic related to nicknames.
+  module Nicknamizable
+    def nickname_length
+      (0...20)
+    end
+
+    def nicknamize(name)
+      disambiguate(name.parameterize(separator: "_")[nickname_length])
```

Comment:
I think you can change `[nickname_length]` to `.first(nickname_length)`, and make the method like this:

```ruby
def nickname_length
  20
end
```